demo/budg/dashboard/src/components/FilesList.js

```diff
@@ -1,164 +1,55 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
-import config from '../meta/server.json'; // eslint-disable-line import/no-unresolved
-
-import handleErrors from '../lib/handleErrors';
-
-const demoServer = `${config.ServiceEndpoint}/demo`;
-
-class FilesList extends Component {
-  constructor() {
-    super();
-
-    this.state = {
-      loading: true,
-      files: [],
-      links: {},
-    };
-
-    this.generateLink = this.generateLink.bind(this);
-    this.deleteFile = this.deleteFile.bind(this);
-    this.loadFiles = this.loadFiles.bind(this);
-  }
-
-  componentDidMount() {
-    this.loadFiles();
```

Comment:
Still called in this component, gets a bit unclear where it comes from if one does not follow along the `List.js` before.